src/main/java/pro/sky/telegrambot/model/notificationTtask.java

```diff
@@ -0,0 +1,53 @@
+package pro.sky.telegrambot.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
+public class notificationTtask {
```

Comment:
классы именуются с большой буквы
```suggestion
public class NotificationTask {
```
https://www.javatpoint.com/java-naming-conventions